Code/Engine/Core/ResourceManager/Resource.h

```diff
@@ -44,7 +44,7 @@ class EZ_CORE_DLL ezResource : public ezReflectedClass
 
   /// \brief Returns the unique ID that identifies this resource. On a file resource this might be a path. Can also be a GUID or any other
   /// scheme that uniquely identifies the resource.
-  EZ_ALWAYS_INLINE const ezString& GetResourceID() const { return m_sUniqueID; }
```

Comment:
Changed the return type here, which is why all previous occurrances had to be adjusted (lots of calls to .GetData() and direct const char* stuff).